tests/Feature/Console/KernelTest.php

```diff
@@ -9,7 +9,7 @@
 const EVERY_MINUTE = '* * * * *';
 
 it('schedules the `scout:index-models` command correctly if enabled', function () {
-    Config::set('explorer.scout.run_jobs', true);
+    Config::set('arkscan.scout.run_jobs', true);
```

Comment:
interesting that this test worked while the config changed 🤔

Reply:
apparently the config check in `Kernel.php` wasn't updated so it referenced an old one with a default value